From fad15b0d2eaacc03213c1369581c91bc93204561 Mon Sep 17 00:00:00 2001
From: "Sascha Kuehndel (InuSasha)" <dev@inusasha.de>
Date: Mon, 18 May 2015 22:05:06 +0200
Subject: [PATCH] [homescreen] update selection of recently added
 movies/tvshows/episodes - movies+tvshows+musicvideos can filter out the seen
 one - tvshows have now 4 selection modes (tvshows with recently added
 episodes, in progress tvshows, in progress and recently added tvshows,
 recently added episodes) - setup over videolibrary settings (defaults to the
 old behavior)

---
 720p/IncludesHomeRecentlyAdded.xml                 |   8 +-
 1 files changed, 250 insertions(+), 48 deletions(-)

diff --git a/720p/IncludesHomeRecentlyAdded.xml b/720p/IncludesHomeRecentlyAdded.xml
index 6496d81..530ab6e 100644
--- a/720p/IncludesHomeRecentlyAdded.xml
+++ b/720p/IncludesHomeRecentlyAdded.xml
@@ -259,7 +259,7 @@
 					<top>220</top>
 					<height>20</height>
 					<width>480</width>
-					<label>20387</label>
+					<label>$INFO[Window.Property(LatestEpisode.Title)]</label>
 					<align>center</align>
 					<aligny>center</aligny>
 					<font>font12_title</font>
@@ -297,7 +297,7 @@
 							<top>10</top>
 							<width>220</width>
 							<height>155</height>
-							<aspectratio>scale</aspectratio>
+							<aspectratio>keep</aspectratio>
 							<texture background="true">$INFO[ListItem.Icon]</texture>
 							<bordertexture border="5">button-nofocus.png</bordertexture>
 							<bordersize>5</bordersize>
@@ -343,7 +343,7 @@
 							<top>10</top>
 							<width>220</width>
 							<height>155</height>
-							<aspectratio>scale</aspectratio>
+							<aspectratio>keep</aspectratio>
 							<texture background="true">$INFO[ListItem.Icon]</texture>
 							<bordertexture border="5">folder-focus.png</bordertexture>
 							<bordersize>5</bordersize>
@@ -354,7 +354,7 @@
 							<top>10</top>
 							<width>220</width>
 							<height>155</height>
-							<aspectratio>scale</aspectratio>
+							<aspectratio>keep</aspectratio>
 							<texture>$INFO[ListItem.Icon]</texture>
 							<bordertexture border="5">button-nofocus.png</bordertexture>
 							<bordersize>5</bordersize>
